### ISSUE TITLE
Handle repeat:Infinity

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/react-formatted-input#5]
Handle repeat when set to Infinity. Perhaps cap it at x characters?

**Comment by alfupe** [comment]
Yes please! At this moment the only way I found doing something similiar is setting a repeat: 1000 or something like that.

Thanks!
